doc/user/content/sql/functions/array_agg.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "array_agg Function"
+description: "Aggregate values (including nulls) as an array."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+The `array_agg(value1, value2, ...)` function aggregates values (including nulls) as an array.
```

Comment:
The function takes a single argument. The multiple values are due to the aggregation, not multiple arguments.